website/docs/r/s3_bucket_server_side_encryption_configuration.html.markdown

```diff
@@ -55,7 +55,7 @@ The `rule` configuration block supports the following arguments:
 
 The `apply_server_side_encryption_by_default` configuration block supports the following arguments:
 
-* `sse_algorithm` - (Required) Server-side encryption algorithm to use. Valid values are `AES256` and `aws:kms`
+* `sse_algorithm` - (Required) Server-side encryption algorithm to use. Valid values are `AES256`, `aws:kms`, and `aws:kms:dsse`
 * `kms_master_key_id` - (Optional) AWS KMS master key ID used for the SSE-KMS encryption. This can only be used when you set the value of `sse_algorithm` as `aws:kms`. The default `aws/s3` AWS KMS master key is used if this element is absent while the `sse_algorithm` is `aws:kms`.
```

Comment:
Shouldn't this be updated as well? `aws:kms:dsse` should require an `kms_master_key_id`, same as `aws:kms`.